DirectEmployers/MyJobs
# Remove pointer from active page

## What's this PR do?
Analytics Tables – Removes pointer cursor from pagination when the page is active. 

## How should this be manually tested?
Analytics > Select enough dimensions for multiple table pages > Switch between pages 
The cursor should not show 'pointer' for the current page. 

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
